Q4-2024 community board blog post (#2262)
writethedocs/www: docs/blog/2024-Q4-community-board.rst

.. post:: Nov 25, 2024
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q4-2024
=======================================================

Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share our final quarterly update for 2024 with you. 

To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

Q4 quarterly meeting highlights
-------------------------------

Our Q4 meeting covered several initiatives that the team leads have been working on, as well as some conference-related topics and areas that we want to focus on in 2025: 

> **The future of jobs and hiring support in the community**:: 

Author: ```suggestion
**The future of jobs and hiring support in the community**
```

>     Following the deprecation of the job board and launch of the new job sponsorship offering (announced in the `October newsletter <https://www.writethedocs.org/blog/newsletter-october-2024/>`_), the WTD organization aims to better align with the changing trends in documentarian jobs and hiring practices. We recognized that having a standalone job board is no longer the best place for connecting our community members with potential jobs, and we hope that the new sponsorship options will provide a better experience for job seekers and hiring organizations. 

Reviewer: ```suggestion
    Following the deprecation of the job board and launch of the new job sponsorship offering (announced in the `October newsletter <https://www.writethedocs.org/blog/newsletter-october-2024/>`_), we aim to better align with the changing trends in documentarian jobs and hiring practices. We recognized that having a standalone job board is no longer the best place for connecting our community members with potential jobs, and we hope that the new sponsorship options will provide a better experience for job seekers and hiring organizations. 
```
I'm not sure why, but "the WTD organization" sounds so weird to me.

    We are also looking at the job-related Slack spaces to make sure that they continue to best serve the community. One of the questions that came up was around moderation of job postings in Slack, and our moderation team is investigating whether there is a need for more strict moderation of these types of posts based on feedback from the community.

> **The future of virtual conferences**::

Author: ```suggestion
**The future of virtual conferences**
```

    Now that another very successful Atlantic conference is behind us, we are starting to think about what the next year(s) will bring for this type of event. What started out in 2020 as a swift response to COVID-19 restrictions became a much-loved part of our community experience, especially for folks who can't travel to in-person events for financial, logistical, or personal reasons. 

    While we intend to keep running virtual events, we also recognize the need to support organizer sustainability and focus on the audience that would benefit the most out of these events. As we start planning next year's virtual event, we'll be reviewing the data from past iterations as well as sourcing feedback from the community so that we can evolve the format and duration of the event. 

> **The future of the WTD website and guides**:: 

Author: ```suggestion
**The future of the WTD website and guides**
```

>     While we'd love to keep running virtual events regularly, we also recognize the need to improve the organizer experience for sustainability, as well as the need to focus on the audience that would benefit the most out of these events. As we start planning next year's virtual event, we'll be reviewing the data from past virtual conferences as well as sourcing feedback from the community and incorporating it into the decision process. 

Reviewer: This sounds to me like dropping virtual events entirely is a serious option we are considering, for 2025 already, which I don't think is the case?

Reviewer 2: Yeah, something like "when planning next years virtual; and physical events we'll be reviewing the data from past virtual conferences as well as sourcing feedback from the community and incorporating it into the decision process."

>     While we'd love to keep running virtual events regularly, we also recognize the need to improve the organizer experience for sustainability, as well as the need to focus on the audience that would benefit the most out of these events. As we start planning next year's virtual event, we'll be reviewing the data from past virtual conferences as well as sourcing feedback from the community and incorporating it into the decision process. 

Author: ```suggestion
    While we intend to keep running virtual events, we also recognize the need to support organizer sustainability and focus on the audience that would benefit the most out of these events. As we start planning next year's virtual event, we'll be reviewing the data from past iterations as well as sourcing feedback from the community so that we can evolve the format and duration of the event. 
```

>     Ever since we launched the documentation guides and resources on our website, we've relied on volunteers to maintain the pages and keep the information up to date. This methodology has had its ups and downs over the years, and recently it has become apparent that the main community website content is in need of more dedicated maintainers. 

Reviewer 2: ```suggestion
    Ever since we launched the documentation guides and resources on our website, we've relied on volunteers to maintain the pages and keep the information up to date. This methodology has had varied results over time, and recently it has become apparent that the main community website content is in need of more dedicated maintainers.    
```

I think it's had varied results, not that the methodology has had ups and downs... 

Also I'm not sure that the outcome is the one we want. Afaik that's one option.

    
    The newsletter team is looking into various options to improve volunteer engagement beyond the newsletter itself, and will be sending out calls to action for folks who might be interested in more long-term involvement as well as possible short-term internship-style contributions. 

> **The future of meetups**::

Author: ```suggestion
**The future of meetups**
```

---------------------------

The `Community Board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community teams.

The board members are team leads for each WTD team, as well as the conference chairs of our various conferences:

* Mikey Ariel (CoreOps, brand and design, board chair)
* Eric Holscher (CoreOps, Portland chair)
* Samuel Wright (CoreOps, infrastructure)
* Sasha Romijn (CoreOps, support, Atlantic chair)
* Rose Williams (meetups)
* Janine Chan (Slack moderation)
* Aaron Collier (newsletter)
* Swapnil Ogale (Australia chair)

The community board meets quarterly in real-time, and provides monthly asynchronous status updates. Board members can share the status of what they're working on and request any help to keep the various teams functioning effectively.

Thank you 
---------

Thanks again to the team leads that participate in the community board, and to all the team members (mostly volunteers!) who work diligently to support the community and each other. If you want to join any of the WTD community teams, check out the `Team page <https://www.writethedocs.org/team/>`_ for team descriptions and contacts. 

> See you in the next update, and wishing everyone a joyous and productive 2024!

Author: ```suggestion
See you in the next update, and wishing everyone a joyous and productive 2025!
```

    While we'd love to keep running virtual events regularly, we also recognize the need to improve the organizer experience for sustainability, as well as the need to focus on the audience that would benefit the most out of these events. As we start planning next year's virtual event, we'll be reviewing the data from past virtual conferences as well as sourcing feedback from the community and incorporating it into the decision process. 

**The future of the WTD website and guides**:: 
    Ever since we launched the documentation guides and resources on our website, we've relied on volunteers to maintain the pages and keep the information up to date. This methodology has had its ups and downs over the years, and recently it has become apparent that the main community website content is in need of more dedicated maintainers. 
    
    The newsletter team is looking into various options to improve volunteer engagement beyond the newsletter itself, and will be sending out calls to action for folks who might be interested in more long-term involvement as well as possible short-term internship-style contributions. 

> 

Author: ```suggestion
```
